test/jit/test_class_type.py

```diff
@@ -483,7 +483,7 @@ def forward(self, a):
 
         # classes are globally registered for now, so we need to clear the JIT
         # registry to simulate loading a new model
-        jit_utils.clear_class_registry()
+        torch.testlib.jit_utils.clear_class_registry()
```

Comment:
Next time, just import `torch.testlib.jit_utils as jit_utils` so you don't have to fix the use sites... (no action needed on this PR)